### DISCUSSION TITLE
Access for collaborators

### DISCUSSION THREAD

**Discussion by kasperwelbers** [avitorio/outstatic#137] [Ideas]
Hi avitorio,

First of all, great work on this package! 

I was wondering whether one of the future goals of outstatic is also to make it possible to let multiple people use the editor. For instance, our use case is that we want to have a website for a research group where teammembers can easily edit. Before I used Netlify CMS (now Decap CMS) for this, but if possible I would love to use Outstatic instead. 

As I understand it, at the moment this is not possible, right? I tried letting a collaborator on the github repo with a nextjs+outstatic site signin via oauth, but he could not see or add collections. Also, even if he could make changes, it wouldnt only trigger a rebuild on vercel if he is a vercel teammember. 

Do you think it would be possible to make this work somehow? For instance if I would add an API token that would let Outstatic make commits under my name? Or using something similar to Git Gateway.

**Comment by avitorio** [comment]
Hey @kasperwelbers ,

Thanks for reaching out and giving outstatic a try.
In fact, if you add someone to the repo and allow them to access the site via the vercel deployment, they should be able to see the contents and edit the site. Also triggering builds.

It's all a matter of setting up the correct environment variables.

Have you added the `OST_REPO_OWNER` and `OST_REPO_SLUG` variable to Vercel?
If not, Outstatic picks up the username of the logged in user and uses it to fetch the repository.

**Comment by kasperwelbers** [comment]
Great! Just tested it and it works.

I thought Vercel only builds if a commit was pushed by the person that owns the Vercel account or a teammember. So glad I was wrong!

Thanks again. I was so not looking forward to maintaining a DB and bucket for some project websites. This really makes my day!

**Reply by avitorio** [reply]
Glad it works for you! We are pushing some awesome new features for v1! Stay tuned 😊
